dbms/src/Flash/Coprocessor/TablesRegionsInfo.cpp

```diff
@@ -47,6 +48,8 @@ const SingleTableRegions & TablesRegionsInfo::getTableRegionInfoByTableID(Int64
 static bool needRemoteRead(const RegionInfo & region_info, const TMTContext & tmt_context)
 {
     fiu_do_on(FailPoints::force_no_local_region_for_mpp_task, { return true; });
+    if (tmt_context.getContext().isDisaggregatedComputeMode())
```

Comment:
add some comments here.
Such as `for xxxx, all region must read from remote xxx`

Reply:
Done